Makefile.am

```diff
@@ -56,6 +56,8 @@ check_PROGRAMS += \
 capn_test_SOURCES = \
 	tests/capn-test.cpp \
 	tests/capn-stream-test.cpp \
+	tests/example-test.cpp \
+	tests/example.capnp.c \
 	compiler/test.capnp.c \
 	compiler/schema-test.cpp \
 	compiler/schema.capnp.c
```

Comment:
Of course, no problem :)

It looks like for some `.capnp` files Make is trying to build `${x}.capnp` from `${x}.capnp.c`, but I haven't dug in further.

Thanks for the repo access.